### ISSUE TITLE
how about multi lstm?

### ISSUE THREAD

**Issue by xinsuinizhuan** [ki-ljl/LSTM-Load-Forecasting#1]
I have the data:
![图片](https://user-images.githubusercontent.com/40679769/158780289-a4ea9c75-ba17-4f6c-88a6-90e8f40f67fa.png)
I want use the pre 10 colums to predict the 11th colums,and want use the multi lstm to forecast,as:
model = Sequential()
layers = [1, 75, 100, prediction_steps]
model.add(LSTM(layers[1], input_shape=(None, layers[0]), return_sequences=True))  # add first layer
model.add(Dropout(0.2))  # add dropout for first layer
model.add(LSTM(layers[2], return_sequences=False))  # add second layer
model.add(Dropout(0.2))  # add dropout for second layer
model.add(Dense(layers[3]))  # add output layer
model.add(Activation('linear'))  # output layer with linear activation
start = time.time()
model.compile(loss="mse", optimizer="rmsprop")
print('Compilation Time : ', time.time() - start)
return model
 how shoul i do?

**Comment by ki-ljl** [comment]
It is recommended that you use PyTorch to build the model. As you said, you want to use the pre 10 columns to predict the 11th column, you can directly modify the nn_seq function in data_process.py.

Like this:
```python
def nn_seq():
    seq = []
    for i in range(len(data)):
        train_seq = []
        train_label = []
        for j in range(0, 10):
            train_seq.append(data[i][j])
        train_label.append(data[i][10])
        train_seq = torch.FloatTensor(train_seq).view(-1)
        train_label = torch.FloatTensor(train_label).view(-1)
        seq.append((train_seq, train_label))
```

**Comment by xinsuinizhuan** [comment]
I am so sorry, I want use the pre 10 rows to predict the 11th rows,  every rows have 96 colums. And i want use the multi lstm, how shoud l do?
